tests/v1/ApiDumper/control-src/DOR/V1/WrittenFile.buphalo.v1.fabrication.yml

```diff
@@ -0,0 +1,21 @@
+preferred_template_trees:
+  - primary
+  - secondary
+actors:
+  '<PrimaryActorName>.php':
+    template: PrimaryActorName.php
+    preferred_template_trees:
+      - primary
+      - secondary
+    annotation_processors:
+      FirstAnnotation:
+        processor_fqcn: \Neighborhoods\Buphalo\V1\AnnotationProcessors\EmptyString
```

Comment:
At first I thought this was something defined for the test, but it's an actual thing in buphalo proper haha, seems like a weird bit of sugar that saves you ~20 characters in very niche circumstances

Reply:
Yeah, it's definitely not super necessary with the SimpleString one, but it's something that can make it clear you're intending to remove an annotation instead of replacing it with a string that you forgot to populate.